### PR TITLE
Create Set-SPOSiteScriptPackage.md

### PR DESCRIPTION
**Pull request by alinsirbu**

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+ [-Description <String>] [-Version <Int32>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Updates a previously uploaded site script package. The package file must be a zip archive containing all the files necessary for the site script. A file called “manifest.json” with script actions must be present in this zip file.
+
+## EXAMPLES
+
+### Example
+This example updates a site script package as a zip file containing a manifest.json script and an updated Dataverse solution zip file. The content of manifest.json is as follows:
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md:26]: Please move the description at the end of the example

Comment (alinsirbu) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md]: @get-itips Done, thanks.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  ],
+  "version": 2
+}
+```
+
+```powershell
+Set-SPOSiteScriptPackage -Identity edaec4ec-71e2-4026-ac1e-6686bb30190e -Title "Install Contoso flow" -Description "Installs the new Contoso flow in a list" -ContentPath "c:\scripts\site-script-package.zip" -Version 2
+```
+
+This example updates a site script package as a zip file containing a manifest.json with script actions as exemplified above and an updated Dataverse solution zip file.
```
[the remark this answers is not in the document]
Comment (alinsirbu) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md:44]: @tseward There is no other documentation that illustrates what exactly should go into manifest.json. Would it make sense for me to re-write the JSON portion as a PS command that writes that blob into a manifest.json file, then add a PS command for obtaining the ZIP archive before the actual main command Set-SPOSiteScriptPackage?

[the remark this answers is not in the document]
Comment (alinsirbu) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md:44]: In the interest of time, I went ahead and implemented the change I suggested. LMK if that works...

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        "listName": "Contoso list",
+        "solutionRelativeFilePath": "solution.zip"
+      }
+  ],
+  "version": 2
+}
+```
+
+```powershell
+Set-SPOSiteScriptPackage -Identity edaec4ec-71e2-4026-ac1e-6686bb30190e -Title "Install Contoso flow" -Description "Installs the new Contoso flow in a list" -ContentPath "c:\scripts\site-script-package.zip" -Version 2
```
[the remark this answers is not in the document]
Comment (alinsirbu) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md:41]: @tseward  I don't see the other articles including any "PS >" prompts in their examples. I don't understand why it's needed...

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+```
+
+This example updates a site script package as a zip file containing a manifest.json with script actions as exemplified above and an updated Dataverse solution zip file.
+
+## PARAMETERS
+
+### -ContentPath
+The path to a zip archive file containing the content of the new site script package.
```
[the remark this answers is not in the document]
Comment (alinsirbu) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md]: absolute path, added. Thanks!

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Version
+A version number of the site script.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md:114]:
```suggestion
The version number of the site script.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteScriptPackage.md:129]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @alinsirbu, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by alinsirbu** [comment]
Looking for approval from @get-itips

**Review by get-itips — APPROVED** [review]

**Pull request merged**